products/cakephp.md

```diff
@@ -23,24 +23,24 @@ releases:
     eol: false
     support: true
     releaseDate: 2022-06-06
-    latest: "4.4.9"
-    latestReleaseDate: 2022-12-29
+    latest: "4.4.10"
+    latestReleaseDate: 2023-01-05
     codename: Strawberry
 
 -   releaseCycle: "4.3"
     eol: true
     support: true
```

Comment:
Should be false ? #2262.

Reply:
That is not correct. 4.3 is still supported with security patches.

Reply:
Then it should not be marked EOL.